Lib/fontTools/feaLib/lexer.py

```diff
@@ -142,13 +142,24 @@ def next_(self):
             return (Lexer.SYMBOL, cur_char, location)
         if cur_char == '"':
             self.pos_ += 1
-            self.scan_until_('"\r\n')
-            if self.pos_ < self.text_length_ and self.text_[self.pos_] == '"':
+            prev_pos = self.pos_
+            strings = []
+            while True:
+                self.scan_until_('"\r\n')
+                if self.pos_ == self.text_length_:
+                    raise FeatureLibError("Expected '\"' to terminate string",
+                                          location)
+                cur_char = self.text_[self.pos_]
                 self.pos_ += 1
-                return (Lexer.STRING, text[start + 1:self.pos_ - 1], location)
-            else:
-                raise FeatureLibError("Expected '\"' to terminate string",
-                                      location)
+                if cur_char == '"':
+                    strings.append(text[prev_pos:self.pos_ - 1])
+                    return (Lexer.STRING, "".join(strings), location)
```

Comment:
I can have a look tomorrow. If you want, you can go on and fix it yourself, no prob ;)